src/main/java/jenkins/plugins/slack/SlackNotifier.java

```diff
@@ -455,6 +461,15 @@ public boolean prebuild(AbstractBuild<?, ?> build, BuildListener listener) {
         return super.prebuild(build, listener);
     }
 
+    @Initializer(after = InitMilestone.JOB_LOADED, before = InitMilestone.COMPLETED)
```

Comment:
Can be migrated with a readResolve, a lot less code and more reliable :smiley:

Reply:
will give it a go 👍

Reply:
Done